diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -125,7 +125,7 @@ data class CommonConfiguration(private val configuration: Map<String, String>?)
      * List of directory names which will be used to detect test sources
      */
     val testAnchors: List<String> by lazy {
-        val testDirs = (configuration ?: emptyMap()).getOrDefault("testDirs", "test").split(',')
+        val testDirs = (configuration ?: emptyMap()).getOrDefault("testDirs", "test").split(',').map { it.trim() }
```

Comment:
maybe update tests to check this change?

Reply:
Added